test/test_lasso_small.jl

```diff
@@ -15,52 +15,68 @@ g = NormL1(lam)
 
 x_star = [-3.877278911564627e-01, 0, 0, 2.174149659863943e-02, 6.168435374149660e-01]
 
-# Nonfast/Nonadaptive
+## Nonfast/Nonadaptive
 
 x0 = zeros(n)
 @time it, x, sol = ProximalAlgorithms.FBS(x0; fq=f, Aq=A, g=g, gamma=1.0/norm(A)^2)
 @test vecnorm(x - x_star, Inf) <= 1e-4
-#@test it == 140
```

Comment:
Rather than eliminating these assertions, why not just relaxing them a bit? Like here put <= 200